frontend/src/metabase/.eslintrc.js

```diff
@@ -0,0 +1,138 @@
+/* eslint-disable import/no-commonjs */
```

Comment:
the only difference is that this block doesn't exist on this branch

```js
{
  files: ["./app.js"],
  rules: {
    "import/no-duplicates": 0,
  },
}
```

so it's been removed, but other than that it looks the same as [the master version.](https://github.com/metabase/metabase/pull/64328/files?diff=split#diff-11cabc1d9ac625d289f0430b563a1e05c04b04c22d9ba194d6e804984b62c763)